<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvBoardMain</class>
 <widget class="QMainWindow" name="AdvBoardMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdvBoardMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,1">
    <item row="0" column="0" colspan="5">
     <widget class="QStackedWidget" name="advBoardWidgets">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>460</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="nextButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="backButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>460</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="openVideoAction"/>
     <addaction name="openSensorDataAction"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
    <addaction name="exitAppAction"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="optionsAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="advStatusBar"/>
  <action name="actionOpen_sensor_data">
   <property name="text">
    <string>Open sensor data</string>
   </property>
  </action>
  <action name="exitAppAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="openVideoAction">
   <property name="text">
    <string>&amp;Video</string>
   </property>
  </action>
  <action name="openSensorDataAction">
   <property name="text">
    <string>Sensor &amp;data</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="optionsAction">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
